03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+LAST_FRAME_NUMBER = 19
+scores = ARGV[0].split(',')
+
+shots = []
```

Comment:
「定数定義」と「プログラム本体のスタート」で区切りたいので、僕だったらこんなふうに改行を入れると思いますー。

```suggestion
LAST_FRAME_NUMBER = 19

scores = ARGV[0].split(',')
shots = []
```

Reply:
改行位置もありがとうございました！修正いたしました。